docs/token-session.md

```diff
@@ -90,22 +90,19 @@ server.route({
   }
 });
 ```
+## Kinto Session Data
 
-## Write to the Session Data
-
-What is required to add a data to a session is the following
-
-1.  have proper apidoc for it
+1.  To add a data to Kinto session have proper apidoc as below.
```

Comment:
You do not need api doc for the kinto session. To use KintoHub's doucmentation for session data, you have to do this apidoc.

We need to clearly describe to set session data and get session data, you must use the headers for requests and responses.